### PR TITLE
Add publications

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

File: db/groups.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ bg:
   aka:
     - bg
     - billinge-group
+  banner: 20151103_173027.png
```
Comment (sbillinge) [inline db/groups.yml]: 👍

**Comment by CJ-Wright** [comment]
This needs a regolith merge and release which is already in PR.

**Comment by CJ-Wright** [comment]
@sbillinge is this ok to merge?

**Comment by sbillinge** [comment]
> @sbillinge is this ok to merge?

This PR seems to include a large publication db but we didn't spend time cleaning fields in that.  Do you want to actually merge that too, and clean after, or do we want to converge the schema for publications and then redo the ingestion?  I am ok either way, but just want to discuss it.

Do you want this merge for the publications db or for the other stuff, which all looks good (a banner in groups.db)

**Comment by CJ-Wright** [comment]
@sbillinge I've pulled the citations for now, I'll do them in another PR. I need to sort out the variables used in the bib file.

**Comment by sbillinge** [comment]
👍 
just to check, you want me to merge now, right?

**Comment by CJ-Wright** [comment]
yes

**Pull request merged**